fares-data-build-tool/fdbt-site
# Feature/tfn 817 additional service day

# Description

Add additional services for service end day and end time

# Testing instructions

Browse http://localhost:5555/multipleProductValidity

- error checking
- hiding and showing the end time column on refresh

Token confirmation should also show the end time if applicable

# Type of change


# Checklist:
